pages/home.js

```diff
@@ -55,8 +84,9 @@ class Home extends React.Component {
   render() {
     function Item({ name, img, price, coffeeBeans }) {
       let beans = [];
-      for (let i = 0; i < coffeeBeans; i++) {
-        beans.push(
+      for (let i = 0; i < 4; i++) {
```

Comment:
why less than 4? arent we doing by 5?, and theres no need to rewrite

Reply:
This is fixed in my PR

Reply:
oh ok got it, just remove the part that I have.